src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/RefAPISource.java

```diff
@@ -109,27 +109,47 @@ public ReferenceBases getReferenceBases(final PipelineOptions pipelineOptions, f
             throw new UserException("Contig " + interval.getContig() + " not in our set of reference names for this reference source");
         }
 
-        try {
-            final Genomics.References.Bases.List listRequest = genomicsService.references().bases().list(referenceNameToIdTable.get(interval.getContig())).setPageSize(pageSize);
-            // We're subtracting 1 with the start but not the end because GA4GH is zero-based (inclusive,exclusive)
-            // for its intervals.
-            listRequest.setStart((long) interval.getGA4GHStart());
-            listRequest.setEnd((long) interval.getGA4GHEnd());
-
-            final ListBasesResponse result = listRequest.execute();
-            if ( result.getSequence() == null ) {
-                throw new UserException("No reference bases returned in query for interval " + interval + ". Is the interval valid for this reference?");
+      try {
+        byte[] bases = null;
+        for (long start = (long) interval.getGA4GHStart();start<(long) interval.getGA4GHEnd();) {
+          final Genomics.References.Bases.List listRequest = genomicsService.references().bases().list(referenceNameToIdTable.get(interval.getContig())).setPageSize(pageSize);
+          listRequest.setStart(start);
+          // if it's more than pageSize, Genomics will clip automatically (so we don't have to)
+          listRequest.setEnd((long) interval.getGA4GHEnd());
```

Comment:
lets change `SimpleInterval.getGA4GHEnd()` and `getGA4GHStart` to return long instead of int since any interop with gat4gh stuff will require logn